components/patient-data/ui/src/main/resources/PhenoTips/PatientSheetCode.xml

```diff
@@ -736,7 +736,10 @@ document.observe('xwiki:dom:loaded', function() {
                }
                newItem.__source = inputElt;
                var tools = new Element('span', {'class' : 'tools hint'});
-               var deleteTool = new Element('span', {'class' : 'tool'}).update("$services.localization.render('phenotips.patientSheetCode.termSuggest.delete')");
+               var deleteTool = new Element('span', {'class' : 'buttonwrapper'}).insert(
+                 new Element('a', {'class' : 'tool button secondary', 'href' : 'javascript:void(0)'}).update("$services.localization.render('phenotips.patientSheetCode.termSuggest.delete')")
```

Comment:
Don't use `javascript:` hrefs, that's a bad practice. If you don't want that to be a link that points somewhere, then don't make it a link.